src/IECoreMaya/FromMayaArrayDataConverter.cpp

```diff
@@ -38,11 +38,33 @@
 
 #include "IECore/VectorTypedData.h"
 
+#include "IECore/GeometricTypedData.h"
+
 using namespace IECore;
 
 namespace IECoreMaya
 {
 
+namespace Detail
```

Comment:
I checked with rigging and it sounds like this feature is used rarely enough that they aren't worried about such a change, so lets go ahead and do it.